package.json

```diff
@@ -1,59 +1,60 @@
 {
-  "name": "eslint-plugin-typescript",
-  "version": "0.14.0",
-  "description": "TypeScript plugin for ESLint",
-  "keywords": [
-    "eslint",
-    "eslintplugin",
-    "eslint-plugin"
-  ],
-  "repository": "nzakas/eslint-plugin-typescript",
-  "author": "Nicholas C. Zakas",
-  "main": "lib/index.js",
-  "scripts": {
-    "lint": "eslint lib/ tests/",
-    "lint:fix": "eslint lib/ tests/ --fix",
-    "docs": "eslint-docs",
-    "docs:check": "eslint-docs check",
-    "format-no-write": "prettier-eslint lib/**/*.js tests/**/*.js --eslint-config-path=.eslintrc --eslint-ignore --prettier-ignore --eslint-path=node_modules/eslint --config=.prettierrc",
-    "format": "yarn format-no-write --write",
-    "format-check": "yarn format-no-write --list-different",
-    "test": "mocha tests --recursive --reporter=dot"
-  },
-  "dependencies": {
-    "requireindex": "^1.2.0",
-    "typescript-eslint-parser": "21.0.2"
-  },
-  "devDependencies": {
-    "eslint": "^5.9.0",
-    "eslint-config-eslint": "^5.0.1",
-    "eslint-config-prettier": "^3.3.0",
-    "eslint-docs": "^0.2.3",
-    "eslint-plugin-eslint-plugin": "^2.0.0",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-prettier": "^3.0.0",
-    "husky": "^1.2.0",
-    "lint-staged": "^8.1.0",
-    "mocha": "^5.2.0",
-    "prettier-eslint-cli": "^4.7.1",
-    "typescript": "~3.1.1"
-  },
-  "peerDependencies": {
-    "eslint": ">=4.13.1 < 6"
-  },
-  "lint-staged": {
-    "*.js": [
-      "yarn format",
-      "git add"
-    ]
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn check --verify-tree && yarn lint && yarn format-check && yarn test && yarn docs:check && lint-staged"
-    }
-  },
-  "engines": {
-    "node": ">=6"
-  },
-  "license": "MIT"
+    "name": "eslint-plugin-typescript",
```

Comment:
package.json file should use 2 spaces instead of 4

Reply:
Does it preserve the 4-space indentation if you install something?

Reply:
editorconfig:

```
# yarn / npm uses 2 spaces when it dumps the file after a change
[package.json]
indent_size = 2

# yml uses 2 spaces
[*.yml]
indent_size = 2
```

Reply:
and entire file is rebuilded/formatted when you install something

Reply:
the latest versions of yarn/npm I believe do preserve the formatting.
It's hard to force contributors to use >= those versions though.

Keeping it at 2 spaces means it's less likely to cause accidental PR diffs.

Reply:
Fixed!